### PR TITLE
BP-1732 Updated the plugin download page for WooCommerce.

### PR DESCRIPTION
**Pull request by harli91**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,99 +1,111 @@
 === Buckaroo Woocommerce Payments Plugin ===
 Contributors: buckaroosupport
 Author: Buckaroo
-Tags: WooCommerce, payments, Buckaroo
+Tags: WooCommerce, payments, Buckaroo, credit card, ideal, bancontact, klarna, sofort, afterpay, giropay
 Requires at least: 4.4.10
 Tested up to: 6.0
 Stable tag: 3.2.3
+Requires PHP: 7.2
```
Comment (Buckaroo-Rens) [inline readme.txt:8]: Are this specs correct? For some reason i'm not sure if it requires at leas 4.4.10 and PHP 7.2.

File: readme.txt [hunk 2]
```diff
@@ -1,99 +1,111 @@
...
-16. Apple Pay
-17. KBC
-18. PayPerEmail
-19. Billink
-20. Belfius
-21. Klarna
+The Buckaroo ([Dutch](https://www.buckaroo.nl/resources/integratie/woocommerce) or [English](https://www.buckaroo.eu/resources/integration/woocommerce) plugin is convenient and works like a charm, it's easy to install and takes all the trouble out of your hands.
+It is a plugin with many different payment methods, all of which are SEPA proof. This plug-in is completely free to download. WooCommerce is an excellent platform for a webshop to look professional, comes with built-in tools to analyze sales and it's also fully customizable. WooCommerce is used by 30% of all webshops worldwide, [download](https://www.buckaroo.nl/integratie/plugins/woocommerce/) this plugin now and find out more!
+
+= Payment methods that are supported in this Buckaroo plugin =
```
Comment (Buckaroo-Rens) [inline readme.txt:19]: @Buckaroo-Rene Did we not talk about setting this methods in alphabetical order?

File: readme.txt (hunk 1 above)
Comment (Buckaroo-Rene) [reply readme.txt:8]: @Buckaroo-Rens,
I've checked this with @harli91 on monday and he confirmed to me that the minimum version was 7.2.

File: readme.txt [hunk 3]
```diff
@@ -1,99 +1,111 @@
...
+1. Centrally manage your payment methods, within the general plugin settings.
+
+2. Linked to the Buckaroo Plaza which is Buckaroo's secured financial backoffice system.
 
 == Frequently Asked Questions ==
 
 = Minimum Requirements =
 - WordPress 4.4
 - WooCommerce 2.2
+- PHP 7.2
```
[the remark this answers is not in the document]
Comment (Buckaroo-Rene) [reply readme.txt:72]: @Buckaroo-Rens,
I've checked this with @harli91 on monday and he confirmed to me that the minimum version was 7.2.

[the remark this answers is not in the document]
Comment (Buckaroo-Rene) [reply readme.txt]: @Buckaroo-Rens , PHP 7.4 is now the minimum as discussed.

**Comment by Buckaroo-Rene** [comment]
@harli91 ,
Can you review this commit?

**Pull request closed without merging**